### ISSUE TITLE
"None" result... bug?

### ISSUE THREAD

**Issue by joel-bitscopic** [cov-lineages/pangolin#425]
Hello,

Pangolin's command line tools as well as the website are giving a "None" result for the attached sequence (renamed ".fasta" to ".fasta.txt"). Nextclade's website is giving a clade (and pango lineage) result. The sequence looks fine upon manual inspection & alignment. 3 amino acid deletions (2 in S, 1 in ORF1a) and a single nucleotide deletion towards the end. 

I thought this may be a "bug" and wanted to report it. I would appreciate any insight you could provide.

Thanks!!

[Pangolin_NONE.fasta.txt](https://github.com/cov-lineages/pangolin/files/8466822/Pangolin_NONE.fasta.txt)

**Comment by AngieHinrichs** [comment]
@joel-bitscopic when I run pangolin on the attached file I get this (in lineage_report.csv):
```
taxon,lineage,conflict,ambiguity_score,scorpio_call,scorpio_support,scorpio_conflict,scorpio_notes,version,pangolin_version,scorpio_version,constellation_version,is_designated,qc_status,qc_notes,note
"PHRL 22 3288/BCRL11.2022-04-06.02/barcode23/ARTIC/medaka MN908947.3 analysis: {""clade"": ""20A"", ""lineage"": ""None"", ""coverage"": ""0.0"", ""meanDepth"": ""0""}",B.1,0.0,,,,,,PUSHER-v1.3,4.0.4,0.3.16,v0.1.4,False,pass,Ambiguous_content:0.02,Usher placements: B.1(1/1)
```
B.1 is assigned (2nd column).  The only "None" that I see comes from the name of the sequence, given in the first line of Pangolin_NONE.fasta.txt as
```
>PHRL 22 3288/BCRL11.2022-04-06.02/barcode23/ARTIC/medaka MN908947.3 analysis: {"clade": "20A", "lineage": "None", "coverage": "0.0", "meanDepth": "0"}
```

If I edit Pangolin_NONE.fasta.txt so that the name is just `PHRL 22 3288/BCRL11.2022-04-06.02/barcode23/ARTIC/medaka` without the inline JSON data, then lineage_report.csv is easier to read:
```
taxon,lineage,conflict,ambiguity_score,scorpio_call,scorpio_support,scorpio_conflict,scorpio_notes,version,pangolin_version,scorpio_version,constellation_version,is_designated,qc_status,qc_notes,note
PHRL 22 3288/BCRL11.2022-04-06.02/barcode23/ARTIC/medaka,B.1,0.0,,,,,,PUSHER-v1.3,4.0.4,0.3.16,v0.1.4,False,pass,Ambiguous_content:0.02,Usher placements: B.1(1/1)
```

**Comment by joel-bitscopic** [comment]
Hello @AngieHinrichs 

Thank you very much for taking the time to respond!

I edited my FASTA header to "none fasta" and re-ran through Pangolin's website and it's still telling me "None" for the lineage. I downloaded the CSV output just to make sure what was showing on the webpage matched the file:

```
Sequence name,Lineage,Conflict,Ambiguity score,Scorpio call,Scorpio support,Scorpio conflict,Note,pangolin version,pangoLEARN version
none fasta,None,,,,,,pangoLEARN lineage assignment AY.43 was not supported by scorpio,3.1.20,2022-02-28

```

I see your output's CSV file is showing different metadata information than the website. I'm guessing you're using command line tools that are a different/newer version. I'm having trouble updating since v4 was released so I'm afraid I will not be able to run this sequence that way just yet. 

Thanks again - I will just simply wait for the current release to stabilize and try the file later.
